tests/test_group.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from random import randrange
+
+import pytest
+import time
+
+from pages.home_page import Home
+
+
+class TestGroup:
+
+    @pytest.mark.credentials
+    def test_group_description_edit(self, base_url, selenium, vouched_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+
+        # Create a new group
+        group_name = (time.strftime('%x-%X'))
+        settings = home_page.header.click_settings_menu_item()
+        group = settings.create_group(group_name)
+
+        # New group data
+        new_group_description = 'This is an automated group.'
+        new_group_irc_channel = '#testgroup'
+
+        # Update the group description fields
+        group_description = group.description.description_info
+        group_description.set_description(new_group_description)
+        group_description.set_irc_channel(new_group_irc_channel)
+        group_description.click_update()
+
+        search_listings = home_page.header.search_for(group_name)
+        group_info = search_listings.open_group(group_name)
+
+        # Check that everything was updated
+        assert new_group_description == group_info.description
+        assert new_group_irc_channel == group_info.irc_channel
+
+        # Delete group
+        group_info.delete_group()
```

Comment:
I could remove it. The only reason I added it there was to "clean up" when possible from random groups each time the tests run.